libs/angular-accelerator/src/lib/components/data-list-grid-sorting/data-list-grid-sorting.component.html

```diff
@@ -1,18 +1,21 @@
 <div class="flex align-items-center gap-2">
-  <p-dropdown
-    id="dataListGridSortingDropdown"
-    [(ngModel)]="selectedSortingOption"
-    class="sort-dropdown"
-    [options]="dropdownOptions"
-    [placeholder]="('OCX_LIST_GRID_SORT.DROPDOWN.PLACEHOLDER' | translate)"
-    (onChange)="selectSorting($event)"
-    [ariaLabel]="('OCX_LIST_GRID_SORT.DROPDOWN.ARIA_LABEL' | translate)"
-  >
-    <ng-template let-item pTemplate="item"> {{ item?.columnName ? (item.columnName | translate) : ''}} </ng-template>
-    <ng-template let-item pTemplate="selectedItem">
-      {{ item?.columnName ? (item.columnName | translate) : ''}}
-    </ng-template></p-dropdown
-  >
+  <p-floatLabel>
+    <p-dropdown
+      id="dataListGridSortingDropdown"
+      [(ngModel)]="selectedSortingOption"
+      class="sort-dropdown"
+      [options]="dropdownOptions"
+      [placeholder]="('OCX_LIST_GRID_SORT.DROPDOWN.PLACEHOLDER' | translate)"
+      (onChange)="selectSorting($event)"
+      [ariaLabel]="('OCX_LIST_GRID_SORT.DROPDOWN.ARIA_LABEL' | translate)"
+    >
+      <ng-template let-item pTemplate="item"> {{ item?.columnName ? (item.columnName | translate) : ''}} </ng-template>
+      <ng-template let-item pTemplate="selectedItem">
+        {{ item?.columnName ? (item.columnName | translate) : ''}}
+      </ng-template></p-dropdown
+    >
+    <label for="dataListGridSortingDropdown">{{ ("OCX_LIST_GRID_SORT.DROPDOWN.ARIA_LABEL" | translate) }}</label>
```

Comment:
Could you please create a separate translation `OCX_LIST_GRID_SORT.DROPDOWN.LABEL` for the floating label? This way, we can change the aria label in the future without unexpectedly modifying the displayed floating label. The new translation key can have the exact same value as the previously used aria label translation.